lib/deppack.js

```diff
@@ -301,6 +301,7 @@ const buildModMap = () => {
 };
 
 const isJs = (fileList, path) => fileList.find(path).type === 'javascript';
+const isTest = (path) => path.indexOf('-test') !== -1;
```

Comment:
This looks very specific to your use case. Is it possible to use `conventions.ignored` for this? If not, what about adding a `config.npm.ignored` anymatch to tell deppack to ignore some app files?